root/defaults/nginx.conf

```diff
@@ -1,53 +1,97 @@
-## Version 2021/02/09 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx.conf
+## Version 2021/04/27 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx.conf
 
 user abc;
-worker_processes 4;
-pid /run/nginx.pid;
+
+# Set number of worker processes automatically based on number of CPU cores.
+worker_processes auto;
```

Comment:
Has this been widely tested? I know it's recommended for bare metal, but does it behave correctly in docker?

Reply:
Changed in https://github.com/linuxserver/docker-swag/pull/115/commits/d77a64aab84cea6cab32847e8e4f91cf3e54906b

We can discuss the new proposed method if needed.